shared/AppInsightsCommon/src/StorageHelperFuncs.ts

```diff
@@ -30,24 +31,55 @@ function _getLocalStorageObject(): Storage {
  * @returns {Storage} Returns storage object verified that it is usable
  */
 function _getVerifiedStorageObject(storageType: StorageType): Storage {
+    let result = null;
```

Comment:
I assume any callers of this function are configured to handle the `returns null` case here?